explanation/intro-to/crypto-libraries.md

```diff
@@ -21,70 +22,64 @@ Among the most popular and widely used libraries and frameworks, we have:
 
 Each one of these has its own implementation details, API, behavior, configuration file, and syntax.
 
-This poses a challenge to system administrators who need to determine what cryptographic algorithms are being used on the systems they deploy. How does one ensure no legacy crypto is being used? Or that no keys below a certain size are ever selected or created? And which types of X509 certificates are acceptable for connecting to remote servers?
+## Determining which libraries are being used by an application
 
-One has to check all of the crypto implementations installed on the system and their configuration. To make things even more complicated, sometimes an application implements its own crypto, without using anything external.
+Ultimately, the only reliable way to determine how an application uses cryptography is by reading its documentation or inspecting its source code. But even then, you may discover that the code is not available and sometimes the documentation might lack this information. If you are having problems finding answers to your crypto-related questions, there are some practical checks that can be made.
 
-## How do we know which library an application is using?
+Generally, the only way to find out these answers about crypto is for the sysadmin to go through a system and find out which crypto implementations are installed on the system and which configurations are being used. To make things even more complicated, sometimes an application implements its own crypto, without using anything external.
```

Comment:
```suggestion
To find out what libraries are being used, one generally needs to check all of the installed crypto implementations and their respective configurations. To make things even more complicated, sometimes an application implements its own crypto, without using anything external.

If you are having problems finding answers to your crypto-related questions, there are some practical checks that can be made.
```

Reply:
I found the original sentence to be a bit circular and confusing - I think this is clearer